assets/src/blocks/ENForm/ENForm.js

```diff
@@ -312,6 +313,9 @@ export class ENForm extends Component {
       if ( undefined === this.props.en_page_id || 0 === this.props.en_page_id ) {
         validationMessage.push( __( '"Engaging Network Live Pages" field is required!', 'planet4-engagingnetworks-backend' ));
       }
+      if ( undefined === this.props.enform_goal || 'not set' === this.props.enform_goal ) {
```

Comment:
I didn't look at what's currently in the database for existing forms (just broke my dev env 😭 ), but if we happen to have an empty `enform_goal` field (so, not undefined or 'not set', just empty string) we should add it to this condition

Reply:
Good point, I will add it 👍